app/src/main/java/org/mozilla/focus/ui/theme/FocusTypography.kt

```diff
@@ -121,4 +122,10 @@ val focusTypography: FocusTypography
             fontSize = 14.sp,
             color = PhotonColors.LightGrey05,
         ),
+        cfrTextStyle = TextStyle(
+            fontFamily = metropolis,
```

Comment:
I think this font should be removed. The other CFRs which don't yet use this typography don't have it declared.
After applying the new patch this start browsing CFR looks different than expected.

![image](https://user-images.githubusercontent.com/11428869/197752462-333a6a94-a245-4fa7-8bdf-1d655082eb82.png)

Reply:
Done